### ISSUE TITLE
Doctrine query result(?) caching saves are failing

### ISSUE THREAD

**Issue by NoelLH** [outlandishideas/wpackagist#415]
At least some attempt by Doctrine to put things on disk is hitting permission errors.

We may be better moving it to ephemeral storage on ECS and/or applying permission hacks to the filesystem it's already trying to work with.

Only web ECS tasks seem to get this.

```
[2021-06-09T07:40:12.255845+00:00] cache.WARNING: Failed to save key "%5BOutlandish%5CWpackagist%5CEntity%5CPackageData%24CLASSMETADATA%5D%5B1%5D" of type Doctrine\ORM\Mapping\ClassMetadata: file_put_contents(/var/www/html/var/cache/prod/pools/nvQdBfIVhL/60c0705c3e6833.48179303): failed to open stream: Permission denied {"key":"%5BOutlandish%5CWpackagist%5CEntity%5CPackageData%24CLASSMETADATA%5D%5B1%5D","exception":"[object] (ErrorException(code: 0): file_put_contents(/var/www/html/var/cache/prod/pools/nvQdBfIVhL/60c0705c3e6833.48179303): failed to open stream: Permission denied at /var/www/html/vendor/symfony/cache/Traits/FilesystemCommonTrait.php:98)","cache-adapter":"Symfony\\Component\\Cache\\Adapter\\PhpFilesAdapter"} []
```

**Comment by NoelLH** [comment]
I've been trying to work out how to update Symfony's general filesystem cache directory cleanly, so we are working with a consistent path target (no env involved) but without much luck so far.

I also suspect that Doctrine's behaviour may have taken a step backwards when I tidied up path change efforts related to Twig's cache, which is off for the moment (#368).

So for now, I have tried in https://github.com/outlandishideas/wpackagist/commit/363c75cc37f1c79eaa174c844dce9afb75d1e451 just putting back a cache path permission hack we had earlier in these iterations (along with an explicit proxy `doctrine orm:generate-proxies` call which didn't work, is redundant and I am now removing).

I think staging is looking healthier so far, apart from an error from the proxies effort, so will remove that, monitor and consider releasing this fix.
